tests/console/mta.pm

```diff
@@ -37,6 +38,7 @@ sub run {
     # test email transmission
     assert_script_run 'echo "FOOBAR123" | mail root';
     assert_script_run 'postqueue -p';
+    sleep(1);
```

Comment:
If something like this is necessary then we have a more generic problem. I wonder what you observed in detail to add this change. Does it really not work to have the two succeeding `assert_script_run/script_run` calls back to back? Maybe something else can be done for synchronisation in between?

Reply:
This pointed out @mloviska already, I didn't want to touch anything I don't need to but I will change this.
So you can't handle 1 second wait, but you will be satisfied with complex bash command that will check the mail queue and wait, right ?

Reply:
yes :)

Reply:
openQA has performance problems since ever, end it does not get beeter. I was testing also `postqueue -f` but still had to wait at least one second, because things do not happen instantly and when openQA is IO loaded it couldl take even more.
btw I didn;t need any of added wait times on my local instance, of course it's not so loaded as osd or ooo.

Reply:
No doubt about that. I just wonder what to do as a long term sustainable solution. Would we add sleep(1) after every call in virtio console? I doubt that. And the previous assert_script_run should only return if it's completed. Doing another postqueue call might be a good choice to ensure the desired end result.